runtime/oti/j9nonbuilder.h

```diff
@@ -2954,6 +2954,11 @@ typedef struct J9Class {
 #define J9INTERFACECLASS_ITABLEMETHODCOUNT(clazz) ((clazz)->totalInstanceSize)
 #define J9INTERFACECLASS_SET_ITABLEMETHODCOUNT(clazz, value) (clazz)->totalInstanceSize = (value)
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+#define J9ARRAYCLASS_SET_STRIDE(clazz, strideLength) *((UDATA*)&(clazz->flattenedClassCache)) = (UDATA) strideLength
+#define J9ARRAYCLASS_GET_STRIDE(clazz) *((UDATA*)&(clazz->flattenedClassCache))
```

Comment:
Better to fetch the value and cast it at the end, rather than casting the source.